j2cl-tasks/src/main/java/com/vertispan/j2cl/build/provided/BytecodeTask.java

```diff
@@ -29,6 +37,8 @@ public class BytecodeTask extends TaskFactory {
     public static final PathMatcher JAVA_BYTECODE = withSuffix(".class");
     public static final PathMatcher NOT_BYTECODE = p -> !JAVA_BYTECODE.matches(p);
 
+    public static final PathMatcher APT_PROCESSOR = p -> p.getFileName().toString().equals("javax.annotation.processing.Processor");
```

Comment:
I think this can be
```suggestion
    public static final PathMatcher APT_PROCESSOR = p -> p.equals(Paths.get("META-INF", "services", "javax.annotation.processing.Processor");
```
or pull the Path constant out to its own private constant, so there's no risk of reading the wrong file in some random dir